scripts/build-wda.js

```diff
@@ -4,24 +4,36 @@ const B = require('bluebird');
 const {Simctl} = require('node-simctl');
 const {getSimulator} = require('appium-ios-simulator');
 const {logger} = require('appium/support');
+const yargs = require('yargs/yargs');
+const {hideBin} = require('yargs/helpers');
 
 const log = logger.getLogger('WDA');
 
-// TODO: allow passing in all the various build params as CLI args
+const argv = yargs(hideBin(process.argv)).options({
+  sdk: {type: 'string', alias: 'v', demandOption: false, describe: 'iOS SDK version to use'},
+  name: {
+    type: 'string',
+    alias: 'd',
+    demandOption: false,
+    describe: 'Name of the iOS simulator to use',
+  },
+}).argv;
+
 async function build() {
-  const [xcodeVersion, platformVersion] = await B.all([
-    xcode.getVersion(true),
-    xcode.getMaxIOSSDK(),
-  ]);
+  let [xcodeVersion, platformVersion] = await B.all([xcode.getVersion(true), xcode.getMaxIOSSDK()]);
+  platformVersion = argv.sdk || platformVersion;
+
   const verifyDevicePresence = (info) => {
     if (!info) {
-      throw new Error(`Cannot find any available iOS ${platformVersion} Simulator on your system`);
+      throw new Error(
+        `Cannot find any available iOS ${platformVersion} ${argv.name || ''} Simulator on your system`,
```

Comment:
consider assigning argv.name to a variable, like deviceName ot simulatorName to make this script more readable